server/fishtest/views.py

```diff
@@ -139,14 +140,50 @@ def login(request):
     return {}
 
 
+def worker_email(worker_name, message, host_url, blocked):
+    owner = worker_name.split("-")[0]
+    wrapped_message = textwrap.fill(message, width=70)
+    body = f"""\
+Hi {owner},
+
+It seems your worker {worker_name} has some issue(s):
```

Comment:
Maybe something like:

Thank you for contributing to the development of Stockfish. Unfortunately, it seems your fishtest worker {worker_name} has some issue(s):

Reply:
I write something like this:

Dear XXX,

  I'm YYY, one of the fishtest maintainers.

Thank you for contributing to the Stockfish/fishtest projects!

Unfortunately, your worker is having many time losses, see the Events log:

https://tests.stockfishchess.org/actions?action=crash_or_time&user=SimonDemel&text=

Feel free to contact me or by email or, better, on the Discord Stockfish server, #fishtest channel.

https://discord.com/invite/awnh2qZfTT

Enjoy your day,